tests/autobahn/test_autobahn.py

```diff
@@ -6,7 +6,8 @@
 
 import pytest
 from pytest import TempPathFactory
-from python_on_whales import DockerException, docker
+
+python_on_whales = pytest.importorskip("python_on_whales")
```

Comment:
One concern with this is that the type checking disappears as this variable will now be `Any`. mypy won't have any way to know this in the python-on-whales module.

Reply:
Elsewhere in our code we have:
```
if TYPE_CHECKING:
    from aiohttp import worker as base_worker
else:
    base_worker = pytest.importorskip("aiohttp.worker")
```

So, something similar should be good.